source/renderer/app/components/staking/stake-pools/StakePoolsRanking.scss

```diff
@@ -0,0 +1,181 @@
+@import '../stakingConfig';
+@import '../../../themes/mixins/link';
+
+.component {
+  @extend %contentBorderAndBackground;
+  margin-top: 20px;
+  padding: 20px;
+
+  .upper {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 20px;
+
+    .selectWallet {
+      :global {
+        .no-value-selected {
+          &.en-selector {
```

Comment:
@yakovkaravelov camel case as is defined above

Reply:
👍